core/src/main/java/org/elasticsearch/http/netty/HttpRequestHandler.java

```diff
@@ -68,9 +65,11 @@ public void messageReceived(ChannelHandlerContext ctx, MessageEvent e) throws Ex
         // when reading, or using a cumalation buffer
         NettyHttpRequest httpRequest = new NettyHttpRequest(request, e.getChannel());
         if (oue != null) {
-            serverTransport.dispatchRequest(httpRequest, new NettyHttpChannel(serverTransport, httpRequest, corsPattern, oue, detailedErrorsEnabled));
+            serverTransport.dispatchRequest(httpRequest,
```

Comment:
I think you can leave it in one line, no need to break after 80 chars or sth (unless I missed that :-)